FranciscoPerez/react-challenge-project-jan-2023
# Fp 2 fix order time format

## Changes
1. Fix order time format to display minute and seconds using a 2-digit format.

## Purpose
Order time does not zero-pad single-digit minute and second values (displays 7 instead of 07).

[Issue #2](https://github.com/Shift3/react-challenge-project-jan-2023/issues/2)

## Approach
Invoke built-in `toLocaleTimeString` date method to force minute and seconds to display as 2-digits.

## Testing Steps
1. Navigate to the "Order Form" page.
2. Select menu item and quantity.
3. Wait until current time has a single digit minute or second (e.g. 12:07:08 PM).
4. Click "Order it!" button to place order.
5. Click "View Orders" nav link.
6. Verify order time uses 2-digit format for each unit.

## Learning
Googled best practices for converting a date/time to a formatted string which led me to Mozilla's documentation.

[Mozilla Documentation - `toLocaleTimeString` method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleTimeString)

Closes #2
